pkg/cmd/server/origin/storage.go

```diff
@@ -333,6 +325,20 @@ func (c OpenshiftAPIConfig) GetRestStorage() (map[schema.GroupVersion]map[string
 		return nil, fmt.Errorf("error building REST storage: %v", err)
 	}
 
+	originAppsClient, err := oappsclient.NewForConfig(c.GenericConfig.LoopbackClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	coreClient, err := kclientset.NewForConfig(c.GenericConfig.LoopbackClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	deployRollbackClient := deployrollback.Client{
+		GRFn: deployrollback.NewRollbackGenerator().GenerateRollback,
```

Comment:
@tnozicka @kargakis reminds me you wanted to get rid of rollback generator, right?